lib/constants/enums.ts

```diff
@@ -138,3 +138,10 @@ export enum DisconnectionReason {
   AuthFailureInvalidSignature = 12,
   WireProtocolErr = 13,
 }
+
+export enum Limits {
```

Comment:
An `enum` doesn't really make sense here, and will break down once we have 2 currencies with the same limit. This should just be a plain object mapping currency (upper case like `BTC`) to these values. You can move it to a `limits.ts` file that just exports that object.